libraries/CMakeLists.txt

```diff
@@ -53,9 +53,11 @@ afr_module_include_dirs(
 file(GLOB afr_modules_csdk "${AFR_MODULES_DIR}/c_sdk/*/*")
 file(GLOB afr_modules_freertos_plus "${AFR_MODULES_DIR}/freertos_plus/*/*")
 file(GLOB afr_modules_abstractions "${AFR_MODULES_DIR}/abstractions/*")
-file(GLOB afr_modules_c_sdk_v4 "${AFR_MODULES_DIR}/c_sdk_v4/*")
+file(GLOB afr_modules_c_sdk_v4 "${AFR_MODULES_DIR}/c_sdk_v4/standard")
 foreach(module IN LISTS afr_modules_csdk afr_modules_freertos_plus afr_modules_abstractions afr_modules_c_sdk_v4)
     if(IS_DIRECTORY "${module}" AND EXISTS "${module}/CMakeLists.txt")
         afr_add_subdirectory("${module}")
     endif()
 endforeach()
+
+afr_add_subdirectory("${AFR_MODULES_DIR}/c_sdk_v4/integration_test")
```

Comment:
please add a new line at the end of file.

Reply:
This change was accidental. Will revert it.